src/components/expense-list-section/expense-list-toolbar/expense-list-toolbar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import "./expense-list-toolbar.css";
+
+interface TabProps {
+  label: string;
+  onClick: () => void;
+  isActive: boolean;
+}
+
+const Tab: React.FC<TabProps> = ({ label, onClick, isActive }) => (
+  <div className={`tab ${isActive ? "active" : ""}`} onClick={onClick}>
+    {label}
+  </div>
+);
+
+interface ExpenseListToolbarProps {
+  toggleSortOrder: () => void;
+}
+
+const ExpenseListToolbar = ({ toggleSortOrder }: ExpenseListToolbarProps) => {
+  const [activeTab, setActiveTab] = useState<string>("Groups");
+
+  const handleTabClick = (label: string) => {
+    setActiveTab(label);
+    console.log(`Active tab is now: ${label}`);
+  };
+
+  return (
+    <header className="expense-list-toolbar">
+      <nav className="tabs">
+        {["Personal", "Groups"].map((tabLabel, index) => (
+          <Tab
+            key={index}
```

Comment:
[key index anti pattern](https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067)